tasks_management/tests/helpers.py

```diff
@@ -0,0 +1,55 @@
+from api_fhir_r4.utils import DbManagerUtils
+from core.forms import User
+from core.services import create_or_update_interactive_user, create_or_update_core_user
+
+
+class LogInHelper:
```

Comment:
I'm just checking on this one - are there any defined helpers coming from the core? It seems like rather basing test utils that should be covered by base modules.

Reply:
It seems that there is no predefined test helper class in the project. However I use: from core.services import create_or_update_interactive_user, create_or_update_core_user to create users.